### PR TITLE
Improve presentation of quest A0C00Y16

### PR DESCRIPTION
**Pull request by mundifex**
Changes:
- only add dialog links to friends 3 and 4 for messages that reference them
- divulge missing person's location using _hidingplace_
- restore delay during quest reward so gold message displays after gratitude

### CODE REVIEW

**Comment by mundifex** [comment]
Linking to #2190

File: Assets/StreamingAssets/Quests/A0C00Y16.txt [hunk 1]
```diff
@@ -292,9 +294,11 @@ _yes_ task:
 
 _givereward_ task:
 	when _yes_ and _S.18_ 
+	start timer _delay_ -- otherwise gold message is displayed before gratitude message, which seems erroneous
```
Comment (Interkarma) [inline Assets/StreamingAssets/Quests/A0C00Y16.txt:297]: Comments can only be placed at the start of a newline. The parser uses string.StartsWith("-") and eliminates the whole line when a - character is present.

Also need to be careful to put a full empty line below message blocks when adding a comment. The empty line is what terminates the message block, and comments will be incorporated if grouped with the other text lines in block.

**Comment by Interkarma** [comment]
@JayH2971 Could I also get your eyes on this one if you have a minute. Thank you mate. :)

File: Assets/StreamingAssets/Quests/A0C00Y16.txt (hunk 1 above)
Comment (mundifex) [reply Assets/StreamingAssets/Quests/A0C00Y16.txt]: I've updated the quest file according to your comments. I have a feeling I bungled things with respect to updating the branch/PR -- please let me know if so.

**Comment by JayH2971** [comment]
> @JayH2971 Could I also get your eyes on this one if you have a minute. Thank you mate. :)

I added a few fixes to Mundifex's branch, so we can merge everything all in one. It looks OK functionally.

**Comment by Interkarma** [comment]
Thank you for the feedback Jay (on both items). :)

I'm releasing a new 0.12 release tomorrow. I'll include some of these fixes so they can get out for public testing.

**Comment by Interkarma** [comment]
@mundifex Are you able to merge Jay's changes to this PR on your branch? I'll merge this once this the two PRs have their powers combined. :)

**Comment by JayH2971** [comment]
I can certainly do that. Make your choice, and I'll work around it.

**Comment by Interkarma** [comment]
Thanks @JayH2971 - Since you're around, I've merged this PR now. If you can please send your changes through when ready I will merge them also. Cheers!

**Review by Dezzj — COMMENTED** [review]
Veiwed

**Pull request merged**